public/javascripts/overlays/admin.js

```diff
@@ -318,7 +318,27 @@ function sendUpdate() {
         'stateRight': window.stateRight
     };
     socket.emit('update overlay', data);
-    toastNotify();
+    toastNotify('Overlay data updated.');
+}
+
+function changeLayout() {
+    var layout = document.getElementsByClassName('selectedLayout')[0];
+    var data = {
+        'layout': layout ? layout.value : 'empty-layout',
+        'background': document.getElementById('layout-background').value
+    };
+
+    socket.emit('change layout', data);
+    toastNotify('Layout updated.');
+}
+
+function obsConnect() {
+    var data = {
+        'address': document.getElementById('obs-address').value,
+        'password': document.getElementById('obs-password').value
```

Comment:
Yeah, I mentioned this below but I don't think we need to pass this stuff through our server.